concordia/admin/__init__.py

```diff
@@ -280,13 +277,13 @@ def save_model(self, request, obj, form, change):
                     res.append(
                         {"Key": obj_version.object_key, "VersionId": obj_version.id}
                     )
-                    # print(res)
 
                 if res != []:
                     bucket.delete_objects(Delete={"Objects": res})
 
         except Exception as error_message:
-            print(error_message)
+            logger.error(error_message)
```

Comment:
Two thoughts: any logger call should have a message explaining the context in case the exception doesn't have enough information to be meaningful in the logs:

```suggestion
            logger.exception(f"Unhandled exception saving {obj}")
```

However, since we're not doing anything else before raising it again, should we even bother catching it instead of letting the caller's normal error handling take care of it? Since it's going into Sentry anyway I'm not sure whether this is giving us enough context to be worth the extra code.

Reply:
updating the code to catch the exception with a better contextual message and then raising it to sentry.